pkgs/tools/networking/ssl-proxy/default.nix

```diff
@@ -4,18 +4,13 @@ buildGoModule rec {
   pname = "ssl-proxy";
   version = "0.2.7";
```

Comment:
version needs to be updated https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
That makes a lot of sense, TIL. Fixed!